frontend/device/kobo/device.lua

```diff
@@ -152,6 +152,8 @@ local Kobo = Generic:extend{
     hasEclipseWfm = no,
     -- Device ships with various hardware revisions under the same device code, requiring automatic hardware detection...
     automagic_sysfs = false,
+    -- Device is a Clara or Libra Colour
+    isRGB = false,
```

Comment:
The naming convention is that isRGB would be a function, i.e., `yes`, while this should be is_rgb since it's only meant for internal use during init.